grpc/test_support/src/test_context.rs

```diff
@@ -53,18 +50,24 @@ impl TestContext {
                 let source_initial_balance = self
                     .maybe_purse_balance(Some(session_transfer_info.source_purse))
                     .expect("source purse balance");
+
+                let proposer_reward_starting_balance = self.inner.get_proposer_purse_balance();
```

Comment:
> I believe proposer has a specific meaning now: it is the validator that is creating a block and submitting a deploy for execution. 

That's true but it's also the person which is receiving the 'transaction fee' of a deploy regardless of the refund percentage. I'm using this value now to reliably measure balance differences (instead of `after = before - gas_cost` it's `after = before - transaction_fee`)

Reply:
I guess that's right.  Maybe proposer is better after all :smile:.